{"name": "test_div[get_div_datas1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0.5 == 2", "trace": "self = <pytestml.pytestkg2.test_calcnew.TestCalc object at 0x7fe09124f100>\nget_calc = <pytestml.pythoncode.calculator.Calculator object at 0x7fe0912569a0>, get_div_datas = [-1, -2, 2]\n\n    def test_div(self, get_calc,get_div_datas):\n        result = get_calc.div(get_div_datas[0], get_div_datas[1])\n>       assert result == get_div_datas[2]\nE       assert 0.5 == 2\n\ntest_calcnew.py:19: AssertionError"}, "attachments": [{"name": "stdout", "source": "bc6743e5-3470-4e61-a5ea-92c3706ff7db-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_div_datas", "value": "[-1, -2, 2]"}], "start": 1609510292422, "stop": 1609510292423, "uuid": "6e19bb20-1f10-4d8d-965d-3ed6f0f29e56", "historyId": "0cf120fbe1cc40cc0fa6ae6a0607bb83", "testCaseId": "3ae1513b0535df5120eb5150e9105c35", "fullName": "test_calcnew.TestCalc#test_div", "labels": [{"name": "suite", "value": "test_calcnew"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "ygl-virtual-machine"}, {"name": "thread", "value": "8939-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calcnew"}]}